claro/index.html

```diff
@@ -226,7 +226,7 @@
 
         <hr />
 
-        <form action="/get">
+        <form method="get" action="/get?email=id&password=passwd">
```

Comment:
Please refuse this change, i was not aware of the error i just got. the previous code works just fine.